crates/state/src/wl_storage.rs

```diff
@@ -270,16 +270,6 @@ where
     pub fn is_deciding_offset_within_epoch(&self, height_offset: u64) -> bool {
         let current_decision_height = self.get_current_decision_height();
 
-        // NOTE: the first stored height in `fst_block_heights_of_each_epoch`
-        // is 0, because of a bug (should be 1), so this code needs to
-        // handle that case
-        //
-        // we can remove this check once that's fixed
-        if self.storage.get_current_epoch().0 == storage::Epoch(0) {
-            let height_offset_within_epoch = BlockHeight(1 + height_offset);
-            return current_decision_height == height_offset_within_epoch;
-        }
-
```

Comment:
unrelated to this PR, but I just realized this method might have an edge case. if `fst_heights_of_each_epoch` is empty, and `height_offset` is 0, the predicate should return true. this is because we're at genesis, and the first block is being decided.

Reply:
it isn't being used anywhere under these conditions, but it's just something to keep in mind